RockWeb/Blocks/Groups/GroupAttendanceDetail.ascx

```diff
@@ -60,11 +60,11 @@
                                         <Rock:RockCheckBox ID="cbMember" runat="server" Checked='<%# Eval("Attended") %>' Text='<%# string.Concat(Eval("MergedTemplate"), " ", Eval("FullName"))%>' />
                                     </ItemTemplate>
                                 </asp:ListView>
-                                <div class="pull-right margin-b-lg">
-                                    <Rock:PersonPicker ID="ppAddPerson" runat="server" CssClass="picker-menu-right" PersonName="Add New Attendee" OnSelectPerson="ppAddPerson_SelectPerson" />
+                                <div class="pull-right margin-b-lg margin-l-md">
+                                    <Rock:PersonPicker ID="ppAddPerson" runat="server" CssClass="picker-menu-right" OnSelectPerson="ppAddPerson_SelectPerson" />
```

Comment:
Hey @nateh777 .  I'm testing this PR now.  I've run into a small layout issue (picker pops up off-screen) because of the **picker-menu-right** css class.  Do you need that class or can I remove it?

Reply:
@nairdo I thought that was already there on the person picker before my changes. I think it would be fine to remove it, I thought that was a class already there.